### DISCUSSION TITLE
IOS app and HTML in Readme

### DISCUSSION THREAD

**Discussion by mstonjek** [dailydotdev/daily#1078] [Feature Requests]
"I am kinda sad that your markdown doesn't support HTML tags like GitHub. I know that this will be a spicy topic, but I would appreciate an app from the App Store on iOS. I think these news are best to read on mobile.

Thank you for reading! 🗡️ 😄"

**Comment by idoshamun** [comment]
Hi @mstonjek,

We would also love to have native mobile apps but our team is too small to support it at the moment. In the meanwhile, you can enjoy our webapp https://app.daily.dev.

As for the markdown, we will add it to the backlog but no promises at this point 🙏

Thanks a lot for the feedback

**Reply by ibragimazizli** [reply]
Hello everyone! I adore daily.dev and have been using it ever since. I see you don't have a mobile application. I would be delighted to work with you to develop a mobile app for you

**Reply by apathized** [reply]
> Hi @mstonjek,
> 
> We would also love to have native mobile apps but our team is too small to support it at the moment. In the meanwhile, you can enjoy our webapp https://app.daily.dev.
> 
> As for the markdown, we will add it to the backlog but no promises at this point 🙏
> 
> Thanks a lot for the feedback

I hope as the site progressively grows and the team alongside it grows; a native iOS app could happen! daily.dev would be a perfect app one day.

**Comment by mstonjek** [comment]
Guess what guys. App is here 🎉
